packages/form/src/form-item.vue

```diff
@@ -99,6 +99,7 @@ export default defineComponent({
       validator: isValidComponentSize,
     },
   },
+  emits: ['validate'],
```

Comment:
It seems form-item was using `elForm.emit` to  emit validate events, shouldn't you add this in `el-form`?
https://github.com/element-plus/element-plus/pull/1272/files#diff-f9321960c6f4f54a1921b6e167fcc4485e88afbbcdedebfd0845e33ddae0dc91R234

Reply:
I have moved emits to `el-form`